libs/docs/src/navigation/2.1.0.yaml

```diff
@@ -12,6 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+- title: Get started
+  links:
+    - title: "Hello world example: odo v3"
```

Comment:
Should we name it something like "Quick start with odo v3" instead? 

Other options are to put the che and odo guides within the Developing with devfiles page or create a new Quick start page for them

Reply:
> Should we name it something like "Quick start with odo v3" instead?
> 
> Other options are to put the che and odo guides within the Developing with devfiles page or create a new Quick start page for them

Yeah I like this title better let's use it, I will also make this change for Eclipse Che too, "Quick start with Eclipse Che"